### PR TITLE
Lookup: Create TestCafe test for Lookup Basics

### PR DESCRIPTION
**Pull request by jdvictoria**

### CODE REVIEW

**Comment by jdvictoria** [comment]
[WG Tests - Lookup](https://trello.com/c/W2WkXjWY/8599-wg-tests-lookup)

File: apps/demos/testing/widgets/lookup/Lookup.test.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+fixture('Lookup.Basics')
+  .page('http://localhost:8080/')
+  .beforeEach(async (t) => {
+    await t.resizeWindow(900, 800);
+  });
+
+runManualTest('Lookup', 'Basics', ['jQuery', 'React', 'Vue', 'Angular'], (test) => {
+  test('Custom Lookup Appearance', async (t) => {
+    const LOOOKUP_BOX_CLASS = 'dx-lookup';
```
Comment (EugeniyKiyashko) [inline apps/demos/testing/widgets/lookup/Lookup.test.js]: Let's move const from test scope.

File: apps/demos/testing/widgets/lookup/Lookup.test.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import { Selector as $ } from 'testcafe';
+import { runManualTest } from '../../../utils/visual-tests/matrix-test-helper';
+
+fixture('Lookup.Basics')
+  .page('http://localhost:8080/')
+  .beforeEach(async (t) => {
```
Comment (EugeniyKiyashko) [inline apps/demos/testing/widgets/lookup/Lookup.test.js]: This the same as here https://github.com/DevExpress/DevExtreme/pull/26891/files#r1521260396

File: apps/demos/testing/widgets/lookup/Lookup.test.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import { Selector as $ } from 'testcafe';
+import { runManualTest } from '../../../utils/visual-tests/matrix-test-helper';
+
+const LOOOKUP_BOX_CLASS = 'dx-lookup';
```
Comment (EugeniyKiyashko) [inline apps/demos/testing/widgets/lookup/Lookup.test.js]:
```suggestion
const LOOKUP_CLASS = 'dx-lookup';
```

**Review by EugeniyKiyashko — APPROVED** [review]

**Review by ksercs — APPROVED** [review]

**Pull request merged**